lib/track.js

```diff
@@ -18,6 +18,15 @@ export function Track(core, cfg) {
   /** @prop {string} windowId An identifier which lasts until the next page refresh. */
   this.windowId = uuid();
 
+  // Paste events
+  this.classifyPaste = cfg.classifyPaste || 
+  function(e) { 
+    return {
+      pan: e.target.hasAttribute('data-rvn-pan'), 
+      sensitive: treeHasAttr(e.target, 'data-rvn-sensitive')
+    };
+  };
```

Comment:
We can remove this default now - but we'll need to check classifyPaste is defined before attempting to cal lit.